scripts/contiv-network-report.sh

```diff
@@ -205,18 +202,18 @@ do
         then
             IF_NAME=$( trim "${IF_NAME_STATUS[0]}" )
             IF_NAMES+=("$IF_NAME")
-        elif [ $( trim "${IF_NAME_STATUS[0]}" ) == "L3" ]
+        elif [ "$( trim "${IF_NAME_STATUS[0]}" )" == "L3" ]
         then
             IF_IP["$IF_NAME"]=$( trim "${IF_NAME_STATUS[1]}" )
         fi
     done
 
     # Get MAC addresses for all interfaces that have an IP address
-    VPP_MAC_ADDR=$( cat "$nn"/"$VPP_MAC_ADDR_FILE" | grep -v "Name" ) || true
+    VPP_MAC_ADDR=$( grep -v "Name " < "$nn"/"$VPP_MAC_ADDR_FILE" ) || true
     if [ -z "$VPP_IP_ADDR" ]
     then
         printf "$NODE_ERR_FORMAT" "$MISSING_NODE" ""
-        echo "Missing or empty hardware address log: '$nn"/"$VPP_MAC_ADDR_FILE'"
+        echo "ERROR: Missing or empty hardware address log: \\'""$nn""/$VPP_MAC_ADDR_FILE\\'"
```

Comment:
All of these extra quotes don't seem to be helping, doing this would be simpler:
```
echo "ERROR: Missing or empty hardware address log: $nn/$VPP_MAC_ADDR_FILE"
```
or
```
echo "ERROR: Missing or empty hardware address log: '$nn/$VPP_MAC_ADDR_FILE'"
```